### PR TITLE
CODE_OF_CONDUCT Uploaded

### LINKED ISSUE #3575: MIssing Code of Conduct
I have been in conversation with various repositories maintainers and one thing that they make sure to have is **Code of Conduct**

As of now, this repository lags a proper CODE OF CONDUCT. It should be designed and Uploaded by someone

### PR DESCRIPTION
**Pull request by swoyam2609**
## :bookmark_tabs: Summary

CODE OF CONDUCT for contributing to the repository uploaded

Resolves #3575 

### :clipboard: Tasks

Make sure you

- [X] :book: have read the [contribution guidelines](https://github.com/mermaid-js/mermaid/blob/develop/CONTRIBUTING.md)
- [ ] :computer: have added unit/e2e tests (if appropriate)
- [X] :bookmark: targeted `develop` branch

### CODE REVIEW

**Comment by swoyam2609** [comment]
@aloisklink Sure, I would add the same and then will make a pull request

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+.
```
Comment (sidharthv96) [inline CODE_OF_CONDUCT.md:63]: image

Missing contact info. You can use `security@mermaid.live` for now.

**Review by sidharthv96 — CHANGES_REQUESTED** [review]

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (swoyam2609) [reply CODE_OF_CONDUCT.md]: As requested, I've added the same, please review

**Review by sidharthv96 — APPROVED** [review]
@knsv please review...

**Review by knsv — APPROVED** [review]

**Comment by knsv** [comment]
This looks great! It mirrors what I think the community is like today and will help us to keep it that way!

**Pull request merged**